builds/e2e/templates/connectivity-deploy.yaml

```diff
@@ -159,3 +160,20 @@ steps:
       E2E_parentHostname: ${{ parameters['connectivity.parentHostname'] }}
       E2E_parentEdgeDevice: ${{ parameters['connectivity.parentEdgeDevice'] }} 
       E2E_hostname: $(Hostname)
+
+  - pwsh: |
+      $logDir = '$(Build.ArtifactStagingDirectory)/logs${{ parameters['networkController.description'] }}'
+      New-Item $logDir -ItemType Directory -Force | Out-Null
+      $artifactSuffix = '$(Build.BuildNumber)-${{ parameters['networkController.description'] }}' -replace '[_|/|(]','-' -replace '[)]',''
+      Copy-Item "$(Agent.HomeDirectory)/../working/support/*.zip" "$logDir/"
+      Write-Output "##vso[task.setvariable variable=artifactSuffix] $artifactSuffix"
+    displayName: "Copy Support Bundle"
+    condition: succeededOrFailed()
```

Comment:
nit:
It might improve readability to add whitespace and change up the ordering:

```
 $artifactSuffix = '$(Build.BuildNumber)-${{ parameters['networkController.description'] }}' -replace '[_|/|(]','-' -replace '[)]',''
Write-Output "##vso[task.setvariable variable=artifactSuffix] $artifactSuffix"

$logDir = '$(Build.ArtifactStagingDirectory)/logs${{ parameters['networkController.description'] }}'
New-Item $logDir -ItemType Directory -Force | Out-Null
 Copy-Item "$(Agent.HomeDirectory)/../working/support/*.zip" "$logDir/"

```